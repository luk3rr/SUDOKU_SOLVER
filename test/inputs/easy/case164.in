500090000 006000230 000004010 007000000 300000190 200053000 053060009 804309600 000510008 
