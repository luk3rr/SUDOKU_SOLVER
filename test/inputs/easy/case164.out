Solving the following grid:
5 0 0 | 0 9 0 | 0 0 0 
0 0 6 | 0 0 0 | 2 3 0 
0 0 0 | 0 0 4 | 0 1 0 
------+-------+------
0 0 7 | 0 0 0 | 0 0 0 
3 0 0 | 0 0 0 | 1 9 0 
2 0 0 | 0 5 3 | 0 0 0 
------+-------+------
0 5 3 | 0 6 0 | 0 0 9 
8 0 4 | 3 0 9 | 6 0 0 
0 0 0 | 5 1 0 | 0 0 8 


Solution found :')

5 3 1 | 7 9 2 | 8 6 4 
4 9 6 | 1 8 5 | 2 3 7 
7 8 2 | 6 3 4 | 9 1 5 
------+-------+------
9 6 7 | 2 4 1 | 5 8 3 
3 4 5 | 8 7 6 | 1 9 2 
2 1 8 | 9 5 3 | 4 7 6 
------+-------+------
1 5 3 | 4 6 8 | 7 2 9 
8 7 4 | 3 2 9 | 6 5 1 
6 2 9 | 5 1 7 | 3 4 8 

Algorithm: BFS
Total time: 96 ms
Total expanded states: 17031
